templates/artist/artist_detail.html

```diff
@@ -167,5 +146,28 @@ <h1 class="header-font">{{ artist.name }}</h1>
             });
         {% endif %}
     </script>
-    <script type="text/javascript" src="{% static "js/artist/artist-detail.js" %}"></script>
+
+    <script>
+        {% if campaign %}
+            var ctx = document.getElementById("myChart5").getContext('2d');
+            var myChart5 = new Chart(ctx, {
+                {% if user_investor %}
```

Comment:
My thought was that it would only show the "Me" section if they were an investor. I must have read this part wrong.